### ISSUE TITLE
Support for ISO 15924: CSV file is zipped

### ISSUE THREAD

**Issue by ajenhl** [kingsdigitallab/django-controlled-vocabulary#12]
ISO 15924 (four letter codes for scripts) is available in CSV (semicolon used as separator) format from the Unicode Consortium at https://www.unicode.org/iso15924/iso15924.txt.zip (linked ultimately from https://www.unicode.org/iso15924/codelists.html). Unfortunately, the file is ZIPped, so none of the existing vocabulary handlers can deal with it. What's the best approach for adding support for this?

**Comment by geoffroy-noel-ddh** [comment]
I can change base_csv.py to support .zip files transparently. 

I'm trying to figure out what the base url should be for that voc. I haven't yet seen examples of authoritative or conventional term uris for it. Also we'll have to make a decision about whether the script code or the number goes into the termid field.

**Comment by geoffroy-noel-ddh** [comment]
I've just sent the question about the URIs to the people in charge of iso 15924.

**Comment by geoffroy-noel-ddh** [comment]
@ajenhl I've added a manager for 15924 and it supports zip files. 

In the absence of any authoritative information about the best prefix & URI scheme for those terms, I've made some arbitrary decisions. E.g. Egyptian hieroglyph has the follwing URI: https://www.unicode.org/iso15924/Egyp (although lookup will fail). And the prefix is iso15924. One possible issue is that the numerc code is not saved in the database.

I hope that'll work for you. Let me know if not. Hopefully the unicode people will also get back to me with useful guidances.
